components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -78,4 +81,77 @@ class ActionSheetTest: XCTestCase {
     }
     
   }
+
+  func testDefaultColors() {
+    // Given
+    let titleTest = "Title"
+
+    // When
+    actionSheet.title = titleTest
+
+    // Then
+    if let title = getLabel(with: titleTest) {
+      XCTAssertEqual(title.textColor, .black)
+    } else {
+      XCTFail("No title label")
+    }
+
+    // Given
+    let messageTest = "Message"
+
+    // When
+    actionSheet.message = messageTest
+
+    // Then
+    if let message = getLabel(with: messageTest) {
+      XCTAssertEqual(message.textColor, .black)
+    } else {
+      XCTFail("No message label was created")
+    }
+  }
+
+  func testColorThemer() {
+    // Given
+    let titleTest = "Title"
+    let messageTest = "Message"
+    actionSheet.title = titleTest
+    actionSheet.message = nil
+    var colorScheme = MDCSemanticColorScheme()
+    colorScheme.onSurfaceColor = .blue
+
+    // When
+    MDCActionSheetColorThemer.applySemanticColorScheme(colorScheme, to: actionSheet)
+
+    // Then
+    if let title = getLabel(with: titleTest) {
+      XCTAssertEqual(title.textColor, colorScheme.onSurfaceColor.withAlphaComponent(0.6))
+    } else {
+      XCTFail("No title label")
+    }
+
+    // Given
```

Comment:
Also looks like a second test is starting here.  Please split these up or combine their sections.